gini/gini-vision-lib-android
# Bugfix protected pdfs

Pdfs with password caused a `SecurityException` when instantiating the `PdfRenderer` on Android API Levels 21+.

Added an additional file import validation when importing pdfs which checks whether the pdf is password protected. Also `SecurityExceptions` are gracefully handled now.

### How to test
A pdf with password and one without can be found in the androidTest assets folder: `ginivision/src/androidTest/assets`. Copy these to a device and open them in one of the example apps. The password protected pdf should cause a validation alert dialog to be shown.
